app/javascript/components/TiptapExtensions/FileEmbedGroup.tsx

```diff
@@ -1,4 +1,5 @@
 import { Node as TiptapNode } from "@tiptap/core";
+import { Node as ProseMirrorNode } from "@tiptap/pm/model";
```

Comment:
I found it a bit confusing that this file is in the "global" `TiptapExtensions` while seemingly being specific to the product edit page, while the (also specific) `FileEmbed` is scoped to that page. Should we move this?

Reply:
Very true, moved!